quickstart/101-azurerm_dns_zone/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "name_servers" {
```

Comment:
If the DNS zone is randomly generated, it would be good to output the value here.

Reply:
Done.